app/src/main/java/com/example/taskthree/viewmodel/CounterViewModel.kt

```diff
@@ -5,17 +5,15 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 
 class CounterViewModel : ViewModel() {
-    private var counter = MutableLiveData<Int>()
+    private val counter = MutableLiveData<Int>()
+    val counterOberser: LiveData<Int>
```

Comment:
Burada bi typo var, onu duzenleyebiliriz. 
```suggestion
    val counterObserver: LiveData<Int>
```